final-progII/src/presentation/Menu.java

```diff
@@ -10,6 +10,9 @@
 import javax.swing.JMenuItem;
 import javax.swing.JPanel;
 
+import logic.ControllerLogic;
+import logic.User;
```

Comment:
Why you import `logic.User` if you are not using it?

Reply:
Maybe I import it for something, but at the end I deleted it.